package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Commonly used functions for our testing library, extends upon 'cucumber-boilerplate'",
   "main": "index.js",
   "scripts": {
-    "start-transformer": "node -r esm index-transformer.js",
+    "start-transformer": "export NODE_ENV=local; export B2C_USERNAME=vmd.vet.test@gmail.com; export B2C_PASSWORD=Pw4supp0rt;node -r esm index-transformer.js",
```

Comment:
Could we move this to the .env file rather than export them here?

Reply:
This shouldn't have actually been committed because this `start-transformer` function is only temporary and there is no settings/local.env file. Gonna remove it.